### PR TITLE
chore(ocamlformat): use the janestreet profile

### PR DESCRIPTION
**Pull request by emillon**

### CODE REVIEW

**Comment by rgrinberg** [comment]
cc @snowleopard. I looked over the diff and it does look pretty close to what we have internally.

**Review by snowleopard — APPROVED** [review]
Many thanks for doing this! That's going to make our lives so much easier when importing/exporting changes.

**Comment by rgrinberg** [comment]
@julow informed us that this fork of ocamlformat is closer to what jst is using: https://github.com/ocaml-ppx/ocamlformat/pull/2314

@emillon could you give this a try and see if it gives us a diff against this? I don't mind using the fork temporarily.

File: otherlibs/chrome-trace/src/chrome_trace.ml [hunk 1]
```diff
@@ -16,20 +16,18 @@ module Timestamp : sig
...
   type t = float
 
   let of_float_seconds x = x
-
   let to_float_seconds x = x
 
   let to_json f =
     let n = int_of_float @@ (f *. 1_000_000.) in
     `Int n
+  ;;
```
Comment (Alizter) [inline otherlibs/chrome-trace/src/chrome_trace.ml:30]: What are these about?

**Comment by Leonidas-from-XIV** [comment]
@Julow also mentioned that the plan is to upstream these in 0.27, so maybe we can use 0.26 in the meantime and update later? I don't know how painful this is for our existing PRs.

**Comment by rgrinberg** [comment]
This PR will unblock me in syncing with upstream. I would lean against tying that work to ocamlformat's release schedule.

**Comment by Alizter** [comment]
Why does jst have to always use janestreet for formatting? Can an exception for dune not be made?

**Comment by rgrinberg** [comment]
We should find ways to get important work done that minimize the use of the limited resources that we have. This PR is a step towards that. Existing policies and their exceptions are of a secondary concern to us.

**Review by Alizter — APPROVED** [review]
Very well, LGTM as long as we add this to the ignoredrevs list too. (after this PR I guess).

**Comment by emillon** [comment]
> @emillon could you give this a try and see if it gives us a diff against this? I don't mind using the fork temporarily.

I added a commit that uses ocaml-ppx/ocamlformat#2314.

> This PR is a step towards that. Existing policies and their exceptions are of a secondary concern to us.

For the part that's about using the janestreet profile, I agree, that's fine. But requiring a fork for development, I don't think that's a good tradeoff to make because this will make contributing harder for everyone.

In terms of diff distance, switching to JS changes about 37kloc. Switching to the fork adds relatively low 4kloc to that. So, just using the profile from the released version makes conflicts 10x less likely for you and changes nothing for contributors. This seems like a good tradeoff. Can we start with this?

**Comment by rgrinberg** [comment]
Sure, let's give it a try. I'll try pulling and see if the conflict resolution is acceptable.

**Comment by snowleopard** [comment]
Thanks everyone!

**Pull request merged**